src/libstd/net/mod.rs

```diff
@@ -15,8 +15,25 @@
 //!   with networking objects like [`TcpListener`], [`TcpStream`] or [`UdpSocket`]
 //! * Other types are return or parameter types for various methods in this module
 //!
+//! # IP addresses helpers stability guarantees
+//!
+//! [`IpAddr`], [`Ipv4Addr`] and [`Ipv6Addr`] offer helper methods that provide information about
+//! an address, for instance whether it is globally routable (see [`IpAddr::is_global()`]), or if
+//! it is a multicast address (see [`IpAddr::is_multicast()`]). These methods are compliant with
+//! the [IETF RFCs]. As erratas and new RFCs are published, these methods behavior may be subject
+//! to changes.
+//!
+//!
+//! For instance, the `240/4` IPv4 block is currently reserved for "future use". If it is made
```

Comment:
I think it's clearer to express this using an IP address.

```suggestion
//! For instance, the `240.0.0.0/4` IPv4 block is currently reserved for "future use". If it is made
```